runtime/jit_vm/cthelpers.cpp

```diff
@@ -78,6 +78,48 @@ jitGetInterfaceITableIndexFromCP(J9VMThread *currentThread, J9ConstantPool *cons
 	return interfaceClass;
 }
 
+/* Only returns non-null if the method is not to be dispatched by itable, i.e.
+ * if it is:
+ * - private, using direct dispatch;
+ * - a final method of Object, using direct dispatch; or
+ * - a non-final method of Object, using virtual dispatch.
+ */
+J9Method*
+jitGetImproperInterfaceMethodFromCP(J9VMThread *currentThread, J9ConstantPool *constantPool, UDATA cpIndex)
+	{
```

Comment:
Nitpick: VM files don't indent the opening and closing function braces.

Reply:
Fixed